rag_experiment_accelerator/doc_loader/pdfLoader.py

```diff
@@ -60,9 +85,13 @@ def load_pdf_files(
 
     logger.info("Loading PDF files")
     documents = []
+    load_pdf_from_path = (
+        _load_pdf_using_pypdf
```

Comment:
Severity: Potential Issue - Consider logging which PDF loading strategy is being used (`_load_pdf_using_pypdf` or `_load_pdf_using_unstructured`). This will provide better traceability for debugging.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.